### ISSUE TITLE
Overblasts Aren't Airdriven?!

### ISSUE THREAD

**Issue by LeahLuong** [Anuken/Mindustry#8144]
**Platform**: Android

**Build**: 140.4, from itch.io

**Issue**: Airblasts don't seem to benefit as much as they should from being overdriven.

**Steps to reproduce**:
![Screenshot_20230109-212630](https://user-images.githubusercontent.com/91638992/211470415-96b47613-65e7-4a5b-b6a8-100bd614edb8.png)
![Screenshot_20230109-212659](https://user-images.githubusercontent.com/91638992/211470434-232c98b5-9810-46b1-a0d3-af8aa2069895.png)
![Screenshot_20230109-212748](https://user-images.githubusercontent.com/91638992/211470455-76897f64-07d5-4426-86e3-16325f4406ff.png)
![Screenshot_20230109-212811](https://user-images.githubusercontent.com/91638992/211470474-418f05ab-cc63-4124-bf88-a267c05e9e1b.png)
![Screenshot_20230109-212823](https://user-images.githubusercontent.com/91638992/211470488-aad3184a-d4bf-488e-a254-8d80e584d178.png)
Both drills are activated simultaneously (along w/ projector) by toggling the pylon & allowed to run awhile. Pics 2 & 3 show their respective, alleged rates. Pics 4 & 5 show how much sand was collected after power was toggled off.

Left drill should've collected ~2.25x as much as the right 1 but only had a handful more. Experiment was repeated w/ a non-boosted projector for a similar result. Experiment was repeated w/ a dome for a similar result. Sand totals were w/in 1 of ea. other or identical in controls (i.e., no power supplied to overdrive). Similar results were achieved by offsetting the vaults 1 tile S & connecting w/ 2 titanium conveyors apiece.

In conclusion, overdrive doesn't seem to be correctly applied to these drills although the graphics seem to indicate it should be, both in the overdriven rates shown to the player & how the drills are highlighted when the projector/dome is selected.

**Link(s) to mod(s) used**: N/A

**Save file**: Please don't reap my soul. IDK how to provide the file & so I've uploaded the pics above. Let me know how & I can try to do it.

**(Crash) logs**: N/A

---
- [ ] **I have updated to the latest release (https://github.com/Anuken/Mindustry/releases) to make sure my issue has not been fixed.**
- [X] **I have searched the closed and open issues to make sure that this problem has not already been reported.**

**Comment by BlueWolf3682** [comment]
![image](https://user-images.githubusercontent.com/61173114/211490233-1095a9ec-afa4-42cf-8c42-5b5dcb1f7f78.png)
![image](https://user-images.githubusercontent.com/61173114/211490269-cefb83dc-5885-4b6c-9509-703685224291.png)
cant reproduce

**Comment by Anuken** [comment]
As the issue template says, don't open issues unless you can provide a save file. I will not be investigating issues unless you can provide one... and this does not seem reproducible anyway.

**Comment by LeahLuong** [comment]
@Anuken I put the .zip in the form above by editing the OP. File was so small compressing actually made it slightly larger  but github wouldn't take it otherwise. My experimental setup should be found near the center of the map if the game doesn't automatically place you there. You can reset the test by replacing the vaults w/ empty 1's & then toggling power.

@BlueWolf3682 Did you test this on the itch.io version using an Android??

**Comment by Anuken** [comment]
![2023-01-10-19-08-45](https://user-images.githubusercontent.com/10100000/211688099-1791c6e8-8915-475d-9137-3335dfaa1534.gif)

Doesn't occur on my machine.

**Comment by LeahLuong** [comment]
@Anuken That's really strange. I've tested this w/ the other drills & many of the other production bldg's from Serpulo; only airblasts seem affected. Other drills, watered or not, display the correct differential after running awhile & factories run faster when overdriven as expected. It's only airblasts which seem to gain a mere token benefit from overdrive, watered or not, even though the overdriven 1's display a faster animation & false drilling rate.

What do you think is going on here?? How can it be fixed?? I'm not sure reinstalling will help since the last time I did that, the game somehow kept all my settings, schems, & campaign progress after uninstalling & then DL'ing the game from a different source (originally, I'd gotten the game from the Play Store but have switched to using itch.io because I was having frequent crashes; game is now more stable but still occasionally poops out). I can give it a try if that's what you recommend but I'd like to hear what you think 1st.

**Comment by Anuken** [comment]
It's possible your device is too weak to run the game at 60FPS, slowing down game logic.

The fix is to get a better device.

**Comment by LeahLuong** [comment]
@Anuken You're right! I usually keep the game @ 10 FPS max to cut down on random crashes. It seems the airblasts work correctly when @ 30 FPS or above; they progressively get worse as FPS goes lower than that. Pretty wacky how only airblasts seem affected by this but perhaps the problem becomes more widespread on more built-out/active maps. I think you should consider raising the min FPS setting & perhaps the sys reqs for your game if devices on the low end can't cut it.

By the way, I appreciate your taking the time to address my issue. Mindustry is an excellently-designed game & I think you've done wonders to program it efficiently so it can run on potatoes like the 1 I have. On top of that, you've magnanimously made the game freely available @ several outlets. I don't think the world appreciates the greatness of people like yourself as much as it should. You have my respect & high regard.

**Comment by Anuken** [comment]
> I think you should consider raising the min FPS setting & perhaps the sys reqs for your game if devices on the low end can't cut it.

System requirements are very vague (nobody can define what "runs at 60fps under most scenarios" requires), and I don't understand what you could mean by "raising min FPS setting" when it is up to **you** to raise it yourself (don't put it at 10 if you want the game to run properly).

**Comment by Anuken** [comment]
The game starts functioning differently at *any* FPS that is below 60. It's just subtle - and no, this inherently cannot be fixed.
